app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -459,7 +473,14 @@ class SamplesView extends React.Component {
   };
 
   handleBulkDownloadModalOpen = () => {
-    this.setState({ bulkDownloadModalOpen: true });
+    const { maxSamplesBulkDownload } = this.context || {};
+    if (this.props.selectedSampleIds.size > parseInt(maxSamplesBulkDownload)) {
+      this.setState({
+        bulkDownloadButtonTempTooltip: `No more than ${maxSamplesBulkDownload} samples allowed in one download.`,
```

Comment:
this might end up trying to compare `> NaN` and being false, right?

Reply:
The value should be set, but I've added a guard just in case.